spec/models/user_spec.rb

```diff
@@ -1,7 +1,27 @@
 require 'rails_helper'
 
 RSpec.describe User, type: :model do
-  
-  it { is_expected.to have_db_column :email }
-  it { is_expected.to have_db_column :encrypted_password }
+
+  user = FactoryBot.create(:user)
+  describe 'Database table' do
+    it { is_expected.to have_db_column :email }
+    it { is_expected.to have_db_column :encrypted_password }
+  end
+
+  describe 'Factory' do
+    it 'can create a valid instance' do
+      expect(user).to be_valid
+    end
+  end
+
+  describe "Validations" do
+    it "is not valid without a email" do
+      user.email = nil
+      expect(user).to_not be_valid
+    end
```

Comment:
Please use ShouldaMatcher matchers to test validations. As I showed you in one of my demos.

Reply:
Changed to Should Matchers, although was working like a butterfly.